manifests/package.pp

```diff
@@ -7,6 +7,7 @@
         ensure  => present,
         require => Class['::elastic_stack::repo']
       }
+      Class['apt::update'] -> Pakcage['filebeat']
```

Comment:
I don't think this works as the resource is incorrect? Pakcage vs Package?

Reply:
Fixed. Thank you for spotting this.